platform/srf06-cc26xx/common/i2c.h

```diff
@@ -99,33 +105,41 @@ bool board_i2c_write_single(uint8_t data);
  * \param rlen Number of bytes to read
  * \return True on success
  */
-bool board_i2c_write_read(uint8_t *wdata, uint8_t wlen, uint8_t *rdata,
+bool i2c_write_read(uint8_t *wdata, uint8_t wlen, uint8_t *rdata,
                           uint8_t rlen);
 
 /**
+ * \brief Write one byte and read len bytes in one operation
+ * \param wdata Pointer to the buffer to be written
+ * \param buf Pointer to the buffer to be written
+ * \param len Number of bytes to write
```

Comment:
Those "param" descriptions are wrong: `wdata` isn't a pointer; `buf` and `len` are for the "read" half of the operation.